driftyco/ionic
# fix(sass): remove usage of colors other than primary, improve error

The best way to test this:
1. Create a starter app
2. Remove all colors from the `$colors` map except for `primary`
3. Run `ionic serve`, see the sass errors
4. Change the theme to `dark` and verify it still errors
5. Update to this branch and link the app to this repo: https://github.com/driftyco/ionic/blob/master/scripts/README.md#developing-against-ionic-locally
6. Repeat the above steps but verify there are no errors.
7. Add a function call to one of the missing colors and check that the error message has improved. Ex: https://github.com/driftyco/ionic-conference-app/blob/master/src/pages/schedule-filter/schedule-filter.scss#L3

fixes #8266
